Watch Extension/FeedingView.swift

```diff
@@ -0,0 +1,33 @@
+import SwiftUI
+
+struct FeedingView: View {
+    @ObservedObject var store: Store<AppState, AppAction>
+    @State var isShowingActionSheet = false
+    let feeding: Feeding
+    var body: some View {
+        Text("00:00:00")
+            .actionSheet(isPresented: $isShowingActionSheet) {
+                ActionSheet(title: Text("What do you want to do?"),
+                            buttons: [
+                                .default(Text("Stop")) {
+                                    self.store.send(.stop(self.feeding))
+                                },
+                                // TODO: make this real
+                                .default(Text("Pause/Resume")) {
+                                    self.store.send(.pause(self.feeding))
+                                }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/9397ad90d7ca569afad13609babe3778b69f0d95/Watch Extension/FeedingView.swift#L18"></span>Multi-line collection literals should have trailing commas.<br />
<code>trailing_comma</code> <code>FeedingView.swift:18</code></td>
    </tr>
  </tbody>
</table>